stock_move_auto_assign/models/product_product.py

```diff
@@ -31,6 +31,10 @@ def _moves_auto_assign_domain(self, locations):
             # out anyway.
         ]
 
+    def _get_move_auto_assign_order(self):
+        # See _run_scheduler() method in stock module
+        return "reservation_date, priority desc, date asc, id asc"
```

Comment:
As we do real time re-reservation, I would drop the reservation_date

```suggestion
        return "priority desc, date asc, id asc"
```